### ISSUE TITLE
Cannot use surrogate pair.

### ISSUE THREAD

**Issue by tuchida** [arton/rjb#68]
```java
Rjb.import('java.lang.System').out.println('𠮷野家')
// => ð ®
```
I want to use emoji in UTF-8. What should I do?

**Comment by arton** [comment]
Rjb 1.5.9 and above does not modify UTF-8 to CESU-8.
https://docs.oracle.com/javase/7/docs/technotes/guides/jni/spec/types.html#wp16542

I'll fix this issue.

**Comment by arton** [comment]
Hi tuchida
Sorry for late response.
I fixed this issue and pushed rjb-1.6.0.gem to rubygem.org.

my only test code is:

**Comment by tuchida** [comment]
Thank you!
I confirmed it work good.
